src/styles/search-results.css

```diff
@@ -0,0 +1,37 @@
+p {
+    text-align: center;
+    font-size: 20px;
+    color: black;
+}
+
+.search-results__image {
+    margin-top: 50px;
+    display: flex;
+    flex-direction: row; 
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    align-items: center;
+    height: 300px;
+    width: 300px;
+}
+
+.search-results {
+    margin: 50px, auto;
```

Comment:
This is erroring out, dont need commas in margin and padding properties like this
```
    margin: 50px auto;
```